test cases/common/1 trivial/meson.build

```diff
@@ -20,7 +20,7 @@ if meson.is_cross_build()
 endif
 
 exe = executable('trivialprog', sources : sources)
-
+assert(exe.name() == 'trivialprog', 'The object name is incorrect')
```

Comment:
It's fine, but just FYI, the 2 argument to assert() is not required any more. It will print `exe.name() == 'trivialprog' assertion failed` automatically, which is just as good.

Reply:
marking as resolved, it doesn't have to be changed ;-)